script_mteb_french/results_analysis/models_characteristics.csv

```diff
@@ -0,0 +1,48 @@
+model,finetuned,multilingual_or_french,number_params,size_gb,seq_len,embedding_dim,model_type,license,tuned_on_sentence_sim
+bert-base-multilingual-cased,0,multilingual,177853440.0,0.71141376,512.0,768.0,pooling,Apache-2.0,0
+bert-base-multilingual-uncased,0,multilingual,167356416.0,0.669425664,512.0,768.0,pooling,Apache-2.0,0
```

Comment:
small error in multilignual_or_french column, I suppose it should be 1 for multilingual models